templates/repo/issue/view_content/comments.tmpl

```diff
@@ -189,5 +189,35 @@
 			</a>
 			<span class="text grey"><a href="{{.Poster.HomeLink}}">{{.Poster.Name}}</a> {{$.i18n.Tr "repo.issues.cancel_tracking_history"  $createdStr | Safe}}</span>
 		</div>
+	{{else if eq .Type 16}}
+		<div class="event">
+			<span class="octicon octicon-primitive-dot"></span>
+			<a class="ui avatar image" href="{{.Poster.HomeLink}}">
+				<img src="{{.Poster.RelAvatarLink}}">
+			</a>
+			<span class="text grey"><a href="{{.Poster.HomeLink}}">{{.Poster.Name}}</a>
+			{{$.i18n.Tr "repo.issues.due_date_added" .Content $createdStr | Safe}}
```

Comment:
Please remove `|Safe` since it is not required (no *safe* HTML content inside)

Reply:
If I do that, it looks like that:
![bildschirmfoto von 2018-04-16 18-47-55](https://user-images.githubusercontent.com/13721712/38823495-cb5b7fe6-41a6-11e8-9e73-10dbfed57aed.png)

So I'm leaving it. (same for the other ones)

Reply:
Fair enough